content/scripts/build.js

```diff
@@ -429,72 +437,99 @@ class Builder {
       !this.options.regenerateAllTitles
     ) {
       // XXX maybe this should become a Map instance.
-      // Or, maybe we can memoize this so that when the server runs runBuild
-      // every time, it's only computed once.
       this.allTitles = JSON.parse(
         fs.readFileSync(allTitlesJsonFilepath, "utf8")
       );
+      return;
     }
 
-    if (!Object.keys(this.allTitles).length) {
```

Comment:
NOTE! The below code **did not change**. It's just that it used to be wrapped in a big `if (!Object.keys(this.allTitles).length) {`  block.